test_data/asm/mem_write_once_external_write.asm

```diff
@@ -1,21 +1,20 @@
-// Very simple write-once memory, but without an mstore operation.
-// As a result, this only works if the content of the `v` column has
+use std::write_once_memory::WriteOnceMemory;
+
+// Uses a simple write-once memory, but without an mstore operation.
+// As a result, this only works if the content of the `value` column has
```

Comment:
does witgen fail or does it just pick a value that works?

Reply:
It fails, because the witness is not unique. However, a different witness generator could pick a value arbitrarily, the constraints allow for that.